astropy/io/fits/header.py

```diff
@@ -1000,12 +985,9 @@ def update(self, *args, **kwargs):
         .. warning::
             As this method works similarly to `dict.update` it is very
             different from the ``Header.update()`` method in PyFITS versions
```

Comment:
Is there a corresponding astropy version? I don't think providing the PyFITS version will help any reader.